react/components/ShippingSimulator/index.js

```diff
@@ -77,7 +83,7 @@ const ShippingSimulator = ({
 
   return (
     <Fragment>
-      <div className={`${styles.shippingContainer} t-small c-on-base`}>
+      <div className={`${styles.shippingContainer} t-small c-on-base`} onKeyPress={keyPress}>
```

Comment:
This event is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Document/keypress_event), you can use [onKeyDown](https://developer.mozilla.org/pt-BR/docs/Web/API/Document/keydown_event) instead